<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-06-16T23:16:18 EDT" name="Default suite" finished-at="2021-06-16T23:16:21 EDT" duration-ms="2720">
    <groups>
    </groups>
    <test started-at="2021-06-16T23:16:18 EDT" name="Default test" finished-at="2021-06-16T23:16:21 EDT" duration-ms="2720">
      <class name="tests.MortgageCalculatorDataDrivenTest">
        <test-method is-config="true" signature="beforeAllTests()[pri:0, instance:tests.MortgageCalculatorDataDrivenTest@1750fbeb]" started-at="2021-06-16T23:16:18 EDT" name="beforeAllTests" finished-at="2021-06-16T23:16:18 EDT" duration-ms="26" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeAllTests -->
        <test-method signature="dataDrivenTests(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:tests.MortgageCalculatorDataDrivenTest@1750fbeb]" started-at="2021-06-16T23:16:18 EDT" name="dataDrivenTests" data-provider="MortgageTestDataSet1" finished-at="2021-06-16T23:16:18 EDT" duration-ms="287" status="FAIL">
          <exception class="org.testng.internal.reflect.MethodMatcherException">
            <message>
              <![CDATA[
[public void tests.MortgageCalculatorDataDrivenTest.dataDrivenTests(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: dataDrivenTests([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=2, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=3, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=4, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=5, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=6, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=7, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=8, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=9, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=10, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) 500000,(java.lang.String) 29,(java.lang.String) 12,(java.lang.String) 30,(java.lang.String) 1,(java.lang.String) Fixed,(java.lang.String) 4.75,(java.lang.String) 2,(java.lang.String) 2021,(java.lang.String) Bi-Weekly,(java.lang.String) $2,608.24,null]]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void tests.MortgageCalculatorDataDrivenTest.dataDrivenTests(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: dataDrivenTests([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=2, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=3, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=4, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=5, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=6, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=7, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=8, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=9, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=10, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) 500000,(java.lang.String) 29,(java.lang.String) 12,(java.lang.String) 30,(java.lang.String) 1,(java.lang.String) Fixed,(java.lang.String) 4.75,(java.lang.String) 2,(java.lang.String) 2021,(java.lang.String) Bi-Weekly,(java.lang.String) $2,608.24,null]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:43)
at org.testng.internal.Parameters.injectParameters(Parameters.java:925)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:34)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.internal.reflect.MethodMatcherException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dataDrivenTests -->
        <test-method is-config="true" signature="afterAllTests()[pri:0, instance:tests.MortgageCalculatorDataDrivenTest@1750fbeb]" started-at="2021-06-16T23:16:18 EDT" name="afterAllTests" finished-at="2021-06-16T23:16:21 EDT" duration-ms="2377" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterAllTests -->
      </class> <!-- tests.MortgageCalculatorDataDrivenTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
